assets/js/admin/admin-scripts.js

```diff
@@ -467,11 +467,43 @@ jQuery.noConflict();
 	var Give_Settings = {
 
 		init: function () {
+			this.setting_change_Country();
```

Comment:
How come the "C" is capped here? Can you just keep them all lowercase?

Reply:
Fixed